DFC.App.JobProfile.CurrentOpportunities/Views/Segment/SimpleList.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@model SimpleListViewModel
+
+<html>
+<body>
+    @if (Model.Items.Any())
+    {
+        <dl>
+            @foreach (var document in Model.Items)
+            {
+                <dt>@document.CanonicalName</dt>
+                <dd>@document.DocumentId</dd>
+            }
+        </dl>
+    }
+</body>
+</html>
```

Comment:
Do we need this? Very similar to the default view.

Reply:
removed as requested, now using "Index" action instead